jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh

```diff
@@ -2,6 +2,17 @@ VERSIONS_REPO_URL="https://github.com/${GITHUB_ORGANIZATION_NAME}/versions.git"
 RELEASE_DATE=$(date +%Y-%m-%d)
 COMMIT_BRANCH="release-${RELEASE_DATE}"
 
+create_pr() {
+    local dest_repo=$1
+
+    # the GITHUB_USER_NAME and GITHUB_PASSWORD variables below refer to
+    # the credentials owner, which is not necessarily the same as the
+    # source repo owner - GITHUB_BOT_NAME in this case
```

Comment:
The text makes it look like that the source repo owner is GITHUB_BOT_NAME, but I think that is the credentials owner, right? I suggest you rephrase it.